store/database/migrate/postgres/0011_update_table_instances_nested_virtualization.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE instances ADD COLUMN IF NOT EXISTS enable_nested_virtualization BOOLEAN NOT NULL DEFAULT false;
```

Comment:
do we need to store in database ? why ?

Reply:
Reason is we need to log during init whether nedtedVirt is enabled or not, for that we are storing nestedVirt variable in instance property and use this instance property to show it in log. For pools > 0 without storing it in db that instance variable is not populating properly (was always showing false, even if nestedVirt is actually enabled in instance). However instance creation was working properly according to the property set in pool.yml

Reply:
add enable_nested_virtualization property in grafana metrics also